client/src/components/Map/LeafletMap.js

```diff
@@ -1,33 +1,48 @@
 import React from "react";
-import { MapContainer, TileLayer, Popup } from "react-leaflet";
+import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
 import styled from "styled-components";
 import "leaflet/dist/leaflet.css";
-import { ReactComponent as MarkerIcon } from "../../assets/marker.svg";
-import Marker from "react-leaflet-enhanced-marker";
 
+// import Marker from "react-leaflet-enhanced-marker";
 import PropTypes from "prop-types";
+import L from "leaflet";
+import icon from "leaflet/dist/images/marker-icon.png";
+import iconShadow from "leaflet/dist/images/marker-shadow.png";
 
 const Map = styled(MapContainer)`
   height: 300px;
   border-radius: 15px;
 `;
 
-const LeafletMap = ({ zoomdistance, mapCenter, markerPosition }) => {
+const LeafletMap = ({
+  zoomdistance,
+  mapCenter,
+  markerPosition,
+  locationName,
+}) => {
+  let DefaultIcon = L.icon({
+    iconUrl: icon,
+    shadowUrl: iconShadow,
+  });
+  L.Marker.prototype.options.icon = DefaultIcon;
+
   return (
-    <Map center={mapCenter} zoom={zoomdistance} scrollWheelZoom={false}>
+    <Map center={mapCenter} zoom={zoomdistance} scrollWheelZoom={"center"}>
+      <Marker position={markerPosition}>
+        <Popup>{locationName}</Popup>
+      </Marker>
+
       <TileLayer
         attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
         url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
```

Comment:
Here the same comment with s, z, x, y it does not really describe what is happening in the url. I am not sure you can change those variables but if you can it could make sense to make the variables more descriptive.

Reply:
I don't think I am able to change those variables since they are specially mentioned in the leaflet documentation [-> Link](https://leafletjs.com/reference-1.7.1.html#tilelayer)